### PR TITLE
Show the last updated aups to the user at aups panel

### PR DESCRIPTION
**Pull request by lionick**

### CODE REVIEW

**Comment by lionick** [comment]
Is related with https://github.com/rciam/comanage-registry/pull/78

File: app/View/CoTermsAndConditions/review.ctp [hunk 1]
```diff
@@ -120,14 +120,13 @@
...
             if(!empty($c['CoTAndCAgreement'])) {
-              print _txt('fd.tc.agree.yes')
-                    . " ("
-                    . $c['CoTAndCAgreement']['identifier']
-                    . ", "
-                    . $this->Time->format($c['CoTAndCAgreement']['agreement_time'], "%c $vv_tz", false, $vv_tz)
-                    . ")";
+              print _txt('fd.tc.agree.yes', array(
+                $c['CoTAndCAgreement']['identifier']
+               ,
```
Comment (ioigoume) [inline app/View/CoTermsAndConditions/review.ctp]: indentation of array

**Review by ioigoume — CHANGES_REQUESTED** [review]

File: app/View/CoTermsAndConditions/review.ctp (hunk 1 above)
Comment (lionick) [reply app/View/CoTermsAndConditions/review.ctp]: done

**Comment by ioigoume** [comment]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**